docs/quality-assurance/manual-tests/Test and document workflow - Reject mentorship relation request1.md

```diff
@@ -0,0 +1,260 @@
+
+##### **EXPECTED RESULT:**
+ Must Show an error- Email already exists.
+##### **ACTUAL RESULT:**
+ email already exists.
+
+* **Test 1.7:-** Enter weak password
+
+<img src = "https://user-images.githubusercontent.com/42871956/93994819-f6ccae00-fdad-11ea-840a-d8307e3b6a90.jpg" width="315" height="620">
+
+##### **EXPECTED RESULT:** 
+Must show an error- password is too weak
+##### **ACTUAL RESULT:**
+ shows password is too weak.
+
+* **Test 1.8:-** Enter different confirm password
+
+<img src = "https://user-images.githubusercontent.com/42871956/93994957-0f3cc880-fdae-11ea-8518-2a3d31662516.jpg" width="315" height="620">
+
+##### **EXPECTED RESULT:** 
+Must show an error- password didn't match
+##### **ACTUAL RESULT:**
+ show an error- password didn't match.
+
+
+
+### *Postive Test Scenarios**
+
+* **Test 1.9:-** Enter valid input in the fields like:-  unique username,email, Strong password must conatin (atlest 1 lower & upper case, digit, special character and longer than 7 characters) 
+
+<img src = "https://user-images.githubusercontent.com/42871956/93995063-2a0f3d00-fdae-11ea-9d3c-fd407935d7b3.gif" width="315" height="620">
+
+##### **EXPECTED RESULT:** 
+Successfull regeatraion/signUp
+##### **ACTUAL RESULT:**
+ Got successfully register in app (shows successful toast)
+ 
+
+### Test 2 - Email Confirmation
+
+After getting registered the confirmation mail is sent to the registered email ID of the user to confirm and then the link will redirect to the confirmation page in a browser 
+
+<img src = "https://user-images.githubusercontent.com/42871956/93995284-70649c00-fdae-11ea-86c8-2f5b2cd0e780.jpg" width="315" height="620"><img src = "https://user-images.githubusercontent.com/42871956/93995303-7490b980-fdae-11ea-865d-014c93d91f48.jpg" width="300" height="620">
+
+##### **EXPECTED RESULT:** 
+Displaying the email confirmation
+##### **ACTUAL RESULT:**
+Displaying Email confirmation but not in proper format
+
+
+
+### Test 3 - Login in app
+
+### *Negative Test Scenarios**
+
+* **Test 3.1:-** Try to login without email verification
+
+<img src="https://user-images.githubusercontent.com/42871956/93995595-d5b88d00-fdae-11ea-8567-b2032313edaf.jpg" width="300" height="620"/>
+
+##### **EXPECTED RESULT:**
+Must show an error- please verify the email before login
+ ##### **ACTUAL RESULT:**
+ please verify the email.</b> 
+
+
+**Test 3.2:-** Enter worng username/email
+
+<img src="https://user-images.githubusercontent.com/42871956/93996347-c2f28800-fdaf-11ea-9e4f-2c5b6f864b6b.jpg" width="300" height="620"/>
+
+##### **EXPECTED RESULT:** 
+User doesn't exists.
+
+##### **ACTUAL RESULT:**
+Something went worng
+
+**Test 3.3:-** Enter worng Password
+
+<img src="https://user-images.githubusercontent.com/42871956/93995797-157f7480-fdaf-11ea-9842-28e5bb36fa85.jpg" width="300" height="620"/>
+
+##### **EXPECTED RESULT:** 
+Worng Password.
+
+##### **ACTUAL RESULT:**
+Something went worng
```

Comment:
@maharishi1 does an issue exist for the above 2 tests where the error message should be more meaningful? If there is no existing issue, could you open one?

Reply:
> @maharishi1 does an issue exist for the above 2 tests where the error message should be more meaningful? If there is no existing issue, could you open one?

No ma'am I didn't see any issue related to this. 
Sure, I am opening an issue.